client/index.html

```diff
@@ -1,9 +1,41 @@
-<!DOCTYPE html>
 <html>
-    <head>
-        <title>Login Page</title>
-    </head>
-    <body>
-        <h1>Hello World!</h1>
-    </body>
-</html>
+<head>
+	<title>
+	Index Login Page
+	</title>
+	
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+	<link rel="stylesheet" type="text/css" href="style.css">
+	
+	<body>
+		
+		<div class = "infobox">
+		
+		<img src="images/user-icon.png" class="icon">
+			<h1> Sign In</h1>
+			<form> 
+			
+				<p> Username</p>
+				<input type = "text" id="username" placeholder="Enter Username">
+				
+				<p>Password</p>
+				<input type = "password" id="pwd" placeholder = "Enter Password">
+				
+				<input type = "submit" value="Login" onclick = "getloginvalues()"/>
+				
+				<a href = "#"> Lost your Password </a> <br>
+				<a href="signup.html"> Don't have an Account? <b>Sign Up Here. </b></a>
+				
+				
+			</form>
+			
+		</div>
+	
+	<script src = "scripts/jquery-3.5.1.js"></script>
+	<script src = "scripts/script.js"> </script>
+		
+	</body>
+	
+	</head>
```

Comment:
```suggestion
```

the head tag typically contains meta information about an html page. The body contains the content. Those are typically separated. The suggested change(s) fix this